03_Glossary_of_Terms.txt

```diff
@@ -83,6 +83,12 @@ Held Actions:
 to a trigger outside of your turn. See "Reactions & Held Actions" in the basic 
 rules document.
 
+IO Ports:
+    Short for Input/Output Ports. Engineer Class Characters can build Drones and
+some Buildings which have a number of IO Ports which can have Peripherals added
```

Comment:
Fairly nitpickish grammar, line 88 should read "some Buildings that have a number of IO Ports, which can have Peripherals added".
The first which is changed to a that, and the second which is preceded by a comma.